### ISSUE TITLE
GRAD CAM issue working with Swin Transformer 

### ISSUE THREAD

**Issue by yathartharora** [jacobgil/pytorch-grad-cam#445]
Hi, 
I have followed the Swin Transformer tutorial but still unable to make it work.

The code doesn't return any error but the it doesn't produce the desired output on the car, sky example as shown in the tutorial.

I think the problem lies in the model_output as its length is not equal to the feature tensor. For some reason the model_output value contains just one tensor value. 

class SimilarityToConceptTarget:
    def __init__(self, features):
        # print("features")
        # print(features.shape)
        self.features = features
    
    def __call__(self, model_output):

        # assert len(model_output)==len(self.features)
        cos = torch.nn.CosineSimilarity(dim=0)
        print("Features Shape",self.features.shape)
        print("Model Output",model_output.shape)
        return cos(model_output, self.features)

I have tried with different target layers still couldn't make it work.
Can you help me with the direction I should explore?
